Hello.java

```diff
@@ -1,7 +1,7 @@
 public class Hello {
 
        Hello(){
-           System.out.print("Hello, World!");
+           System.out.print("Hello, Kara!");
        }
        public static void main(String[] args) {
             Hello helloWorld = new Hello();
```

Comment:
I ended up refactoring the code in a later issue so that should take care of this mistake.